src/xrpld/app/tx/detail/SetAccount.cpp

```diff
@@ -650,6 +650,15 @@ SetAccount::doApply()
             uFlagsOut &= ~lsfDisallowIncomingTrustline;
     }
 
+    // Set or clear flags for disallowing escrow
```

Comment:
`asfAllowTokenLocking` should be checked in preflight?

Other flags don't seem to be checked in preflight...

Reply:
Yeah I just used existing implementation for this